app/services/preserved_object_handler.rb

```diff
@@ -111,17 +157,13 @@ def result_code_msg(response_code, addl=nil)
   end
 
   def result_msg_prefix
-    @msg_prefix ||= "PreservedObjectHandler(#{druid}, #{incoming_version}, #{incoming_size})"
+    @msg_prefix ||= "PreservedObjectHandler(#{druid}, #{incoming_version}, #{incoming_size}, #{storage_dir})"
   end
 
-  # results = [result1, result2]
-  # result1 = {response_code => msg}
-  # result2 = {response_code => msg}
```

Comment:
i think the format of the results argument isn't inherently obvious, which is why I put in these comments.  Any reaction?

Reply:
Yeah I think it is a good idea to add the format of results back. Will do that